frontend/app/src/locales/en.json

```diff
@@ -2261,7 +2261,8 @@
       "api_key": "API Key",
       "api_secret": "API Secret",
       "passphrase": "Passphrase",
-      "kraken_account": "Select the type of your Kraken account"
+      "kraken_account": "Select the type of your Kraken account",
+      "ftx_subaccount": "FTX subaccount name"
```

Comment:
Try to make it clear that this is optional here. Something like "Optionally limit to an FTX subaccount".

Because from this menu it appears as if it's needed.

![2021-06-14_18-43](https://user-images.githubusercontent.com/1658405/121928504-82bda600-cd40-11eb-9d97-bd65a1d6b5d3.png)